### PR TITLE
replace $ and $j with jq

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: configuration/pih/scripts/global/ces.js [hunk 1]
```diff
@@ -27,11 +27,11 @@ function setUpCholesterolSection() {
       dmSection.hide();
     }
   };
-  $(dmCheckbox).change(function () {
```
Comment (mogoodrich) [inline configuration/pih/scripts/global/ces.js:30]: "$" isn't necessarily available in the global context, but jq is...  I went through and did a global search-and-replace on all our forms...

File: configuration/pih/htmlforms/consult.xml [hunk 2]
```diff
@@ -207,19 +207,12 @@
...
         }
 
     </style>
 
-    <script>
-        // I don't really understand why this assignment `$j = jq` is required, but for some
-        // reason a bunch of Javascript gets included in the page that expects `$j` to exist,
-        // but doesn't define `$j` itself. *shrug emoji*
-        var $j = jQuery;
-    </script>
```
Comment (mogoodrich) [inline configuration/pih/htmlforms/consult.xml:221]: I think I've fixed this so this is no longer needed... the Medication section was likely failing before if this wasn't present... I removed this and tested the Medication section locally and it worked... @brandones  do you happen to remember what else was failing so we can test?

Comment (brandones) [reply configuration/pih/htmlforms/consult.xml:221]: I don't, no.

**Review by brandones — APPROVED** [review]
@Joseph-Developer  por favor echar un vistaso. Y necesitamos probar este formulario despues que hacemos merge.

**Comment by mogoodrich** [comment]
Thanks @brandones , merging this in!

**Pull request merged**